### ISSUE TITLE
Fix to Stable 1 Issue Simulation Autocrafting

### ISSUE THREAD

**Issue by desagas** [AppliedEnergistics/Applied-Energistics-2#682]
So, after reading several issues on the AE2 Autocrafting saying "simulation" with the "start" button greyed out, I have found a solution, at least to my world.

I play on modsauce 1.0.10 right now, and for the longest time, my autocrafting would not craft 95% of the time, and would get stuck on the issue mentioned above, and on several other posts. I searched my massive build, every little detail of my system, over and over again, checking p2p tunnels, interface connections, etc. But nothing was out of place, everything was set up perfectly.

Eventually, I got so frustrated, I started looking for duplicates. AE2 has the option to have multiple storage buses for the exact same item. In my case, sand. I had two deep stores from MFR storing sand. One was for constant production while the other was from my ender quarry. One was priority 10 and the other priority 20. I thought, what the heck, lets remove one of the buses, and sure enough, ever since then, 100% of the time, I have not been stopped by the simulation mode.

So in short: if your system is stuck on simulation with the start button stuck as grey, then look for double storage spaces for an item, it can completely ruin your AE2 and autocrafting experience.

**Comment by thatsIch** [comment]
mh I guess this could be also a reason.

Since the network has virtually more items than it can actually request for the crafting.

Thanks for the effort!

**Comment by ignisthedragon** [comment]
Interessting. I can confirm this solution. After I removed all the double provided deep stoages and barrels (from jabba) it became better but some things still didnt work, for example the shapeless crafting of one flint with three gravel (iguna tweaks). I always did stuck at simulation although I changed the priority of the storage bus and configured it to only handle gravel.  So I removed the deep storage for gravel entirely and put some gravel in a storage grate (from better storage). After this there wasnt any problems anymore with crafting jobs which contains gravel. I think the problem comes with the combination of a storagebus and a deep storage (maybe barrels too). Ive tested with the RC2 beta, where the crafting problem still exists.

**Comment by Cisien** [comment]
Recipes that are recursive (charcoal + logs to make more charcoal for example) will stall simulation. 

Both jabba and dsu's use the same 'dsu' API, so it is likely they will both experience the issue

**Comment by bluefi** [comment]
I had the same problem with every recipe i tried. Even crafting 4 Strings into wool didn't work. The Start button was greyed out. But only recipes using the Molecular Assembler were affected. Recipes for other machines like for example the Induction Furnace worked. 

But i found out it wasn't a pure AE2 mistake. It was my fault updating NEI and forgetting to update WAILA. Removing WAILA from the Server fixed it (And probably updating WAILA would have too).


Just if someone is reading this because he has the same problem and made the same mistake as me.
